frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -281,6 +281,146 @@ describe('NewRunParametersV2', () => {
     });
   });
 
+  it('test convertInput function for LIST type with default value', () => {
+    const handleParameterChangeSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        listParam: {
+          parameterType: ParameterType_ParameterTypeEnum.LIST,
+          defaultValue: [1, 2, 3],
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: handleParameterChangeSpy,
+    };
+    render(<NewRunParametersV2 {...props}></NewRunParametersV2>);
+
+    const listParam = screen.getByDisplayValue('[1,2,3]');
+    fireEvent.change(listParam, { target: { value: '[4,5,6]' } });
```

Comment:
Can we check the screen now has [4,5,6]?

`screen.getByDisplayValue('[4,5,6]');`

Reply:
Same for all cases below.